src/test/java/com/exasol/exaoperation/plugin/PluginTest.java

```diff
@@ -22,10 +22,17 @@ void beforeEach() {
     }
 
     @Test
-    void testGetNameFromPackage() {
+    void testGetName() {
         assertThat(this.pluginPackage.getName(), equalTo(PLUGIN_NAME));
     }
 
+    @Test
+    void testGetNameWithDash() {
```

Comment:
Maybe we should make this Parameterized and add all plugins that Exasol is providing today?